facebook_connector.py

```diff
@@ -45,8 +53,23 @@ def resolve_user_id(user_id):
     Returns:
         The name of the user if it is able to find it, otherwise the input
     """
-    # TODO (neitsch): This method is not implemented yet.  Use the user_id
-    # (format: 12345678@facebook.com) to get the actual name of the user
+    fb_id_pattern_match = _FB_ID_PATTERN.match(user_id)
+    if fb_id_pattern_match:
```

Comment:
Use the "guard" approach where you break out as early as possible, e.g.
```
if not fb_id_pattern_match
   return
```

You can then omit the `else` since you're returning and un-indent the code.